Kconfig.zephyr

```diff
@@ -321,6 +321,14 @@ config COMPILER_COLOR_DIAGNOSTICS
 	help
 	  Compiler diagnostic messages are colorized.
 
+config COMPILER_SECURITY_FORTIFY
+	bool "Detect buffer overflows in libc calls"
+	default n
```

Comment:
`default n` is redundant and the repo-wide convention is to omit them.

Reply:
Sounds good.